bin/pping.py

```diff
@@ -154,44 +163,44 @@ def generateEvents(self):
                              Event.UP
                              )
             self.db.new_event(newEvent)
-            debug.debug("%s marked as up." % netbox)
+            LOGGER.info("%s marked as up.", netbox)
 
     def main(self):
         """
         Loops until SIGTERM is caught.
         """
         self.db.start()
         while self._isrunning:
-            debug.debug("Starts pinging....", 7)
+            LOGGER.debug("Starts pinging....")
             self.updateHostList()
             elapsedtime = self.pinger.ping()
             self.generateEvents()
-            debug.debug("%i hosts checked in %03.3f secs. %i hosts "
-                        "currently marked as down." %
-                        (len(self.netboxmap), elapsedtime, len(self.down)))
+            LOGGER.info("%i hosts checked in %03.3f secs. %i hosts "
+                        "currently marked as down.",
+                        len(self.netboxmap), elapsedtime, len(self.down))
             wait = self._looptime-elapsedtime
             if wait > 0:
-                debug.debug("Sleeping %03.3f secs" % wait, 6)
+                LOGGER.info("Sleeping %03.3f secs", wait)
             else:
                 wait = abs(self._looptime + wait)
-                debug.debug("Check lasted longer than looptime. "
-                            "Delaying next check for %03.3f secs" % wait, 2)
+                LOGGER.critical("Check lasted longer than looptime. "
+                                "Delaying next check for %03.3f secs", wait)
             sleep(wait)
 
     def signalhandler(self, signum, frame):
         if signum == signal.SIGTERM:
-            debug.debug("Caught SIGTERM. Exiting.", 1)
+            LOGGER.critical("Caught SIGTERM. Exiting.")
             sys.exit(0)
         elif signum == signal.SIGHUP:
             # reopen the logfile
             logfile_path = self.config.get("logfile", "pping.log")
-            debug.debug("Caught SIGHUP. Reopening logfile...")
+            LOGGER.info("Caught SIGHUP. Reopening logfile...")
```

Comment:
I think we could potentially bake log-reopening into the the library as well; it would be useful for all the NAV daemons....